.github/workflows/build.yaml

```
@@ -38,6 +38,9 @@ jobs:
         uses: DeterminateSystems/flakehub-cache-action@main
 
       - name: Build and cache dev shell for ${{ matrix.systems.nix-system }} on ${{ matrix.systems.runner }}
+        # We still support this system but caching the dev shell fails due to system support mismatch,
+        # and we don't really need this cached anyway
+        if: ${{ matrix.systems.nix-system }} != 'x86_64-darwin'
```

actionlint (1.7.7)

if: condition "${{ matrix.systems.nix-system }} != 'x86_64-darwin'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)